Source/Common/compression.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#if !HC_NOZLIB
+#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK
+#pragma once
+
+#include "utils.h"
+#include <zlib.h>
+
+NAMESPACE_XBOX_HTTP_CLIENT_BEGIN
+
+class Compression
+{
+public:
+    static void Compression::CompressToGzip(uint8_t* inData, uInt inDataSize, HCCompressionLevel compressionLevel, http_internal_vector<uint8_t>& outData);
```

Comment:
>Compression:: [](http://example.com/codeflow?start=16&length=13)

intentional?

Reply:
oh no... that was a copy paste mistake from the source file! Let me correct it

Reply:
Fixed